app/serializers/npq_participant_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+class NPQParticipantSerializer
+  include JSONAPI::Serializer
+  include JSONAPI::Serializer::Instrumentation
+
+  set_id :id
+  set_type :'npq-participant'
+
+  attributes :participant_id, :date_of_birth, :teacher_reference_number, :teacher_reference_number_verified, :active_alert,
+             :school_urn, :school_ukprn, :headteacher_status, :eligible_for_funding, :funding_choice
```

Comment:
How many of those things are the LPs interested in?

Are we missing some big things here too? Like... full name and email?

Reply:
course_identifier will definitely need to be returned if they want to make declarations based on the list returned.